types/runtime.ts

```diff
@@ -12,6 +12,8 @@ export type FunctionConfig = {
   memory?: number;
   retryOnFailure?: boolean;
   staticIp?: boolean;
+  maxInstances?: number | undefined;
```

Comment:
no need to add `| undefined` at the end. The question mark after the key means that the property could be undefined